src/Views/GridView.vala

```diff
@@ -189,4 +191,15 @@ public class Slingshot.Widgets.Grid : Gtk.Grid {
     public void top_left_focus () {
         set_paginated_focus (0, 0);
     }
+
+    public bool on_key_press (Gdk.EventKey event) {
+        switch (event.keyval) {
+            case Gdk.Key.Home:
+            case Gdk.Key.KP_Home:
+                stack.set_visible_child_name ("1");
+                break;
```

Comment:
I wonder why we don't put a `return Gdk.EVENT_STOP` here as we are handling the event :thinking:

Reply:
Good call!